js/src/cli-util/pull.ts

```diff
@@ -219,7 +219,7 @@ function makeFunctionDefinition({
   const promptContents =
     prompt.type === "completion"
       ? `prompt: ${doubleQuote(prompt.content)}`
-      : `messages: ${util.inspect(prompt.messages, { depth: null }).trimStart()}`;
+      : `messages: ${util.inspect(prompt.messages, { depth: null, maxStringLength: Infinity }).trimStart()}`;
```

Comment:
Curious why we use inspect vs json.stringify

Reply:
I can't remember, but I'm sure I started with `stringify` 😓